### PR TITLE
.json sanitize

### PR DESCRIPTION
**Pull request by Xackii**
## About The Pull Request

Made tabs in json files instead of spaces. Also removed space at the end of words in chef Italian language due to which these words were not processed.

## Why It's Good For The Game

Reading this files on github is really hard.

### CODE REVIEW

**Comment by san7890** [comment]
Would be nice if we could lint for this as well

**Comment by Artemchik542** [comment]
Json refactor. Looks like a big deal.
big

File: strings/italian_replacement.json [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
-            "prosciutto"
-        ],
-        "mom": "mamma",
-        "my": "my-a",
-        "nuke": "spiciest-a meatball",
-        "op ": "greek ",
-        "operative": "greek",
-        "operatives": "greeks",
-        "ops": "greeks",
-        "sec ": "polizia ",
```
Comment (SmArtKar) [inline strings/italian_replacement.json:49]: Bring these back - its important to have those spaces as instead things like "flopped" will turn into "flgreekped"

**Review by SmArtKar — CHANGES_REQUESTED** [review]

File: strings/italian_replacement.json (hunk 1 above)
Comment (Xackii) [reply strings/italian_replacement.json:49]: returned it as it was

**Comment by jlsnow301** [comment]
> Would be nice if we could lint for this as well (and also add it to the vscode settings for all new jsons to be automatically be tab delinated)

Prettier can do it

**Review by Ghommie — APPROVED** [review]

**Pull request merged**